virtual_ecosystem/models/hydrology/constants.py

```diff
@@ -111,8 +111,8 @@ class HydroConsts(ConstantsDataclass):
     texture.
     """
 
-    water_retention_curvature: float = -7.22
-    """Curvature of the water retention curve.
+    campbell_pore_size_distribution: float = -7.22
+    """Curvature of the water retention curve as indicator of pore size distribution.
```

Comment:
What does Campbell mean here, is it from a specific model?

Reply:
I added the reference to the constants; the reference is also in the function docstring.